### PR TITLE
Update author details

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose
1. All full affiliations
2. Add updates given by authors in [this](https://docs.google.com/spreadsheets/d/1rtCDjNECWDJmsmAAN4AdCdAIUicwRa4-UmSoZLCUD_Q/edit#gid=1182772523) google sheet.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
@jharenza, can you also add a footnote to indicate that you are the Lead Contact listed in the Resource Availability section, please?

**Comment by jharenza** [comment]
> @jharenza, can you also add a footnote to indicate that you are the Lead Contact listed in the Resource Availability section, please?


Do you mean within the YAML file?

**Comment by jaclyn-taroni** [comment]
Based on the [Final File Checklist](https://www.cell.com/pb-assets/journals/EM/MasterFFCs/XGENFFC.pdf), I believe we need to do two things:

* Add a symbol of some kind in the YAML, e.g., `symbol_str: †`, for your entry.
* Add to [the Contact information section in the front matter](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/9418610a181fb8ab86649f120be5babf392e46a2/content/00.front-matter.md#contact-information): 

```
<sup>†</sup>: Lead Contact.
```

**Comment by jaclyn-taroni** [comment]
Based on that failure, I might try a different character. It didn't seem like there was trailing whitespace, but that's another idea I had.

File: content/metadata.yaml [hunk 1]
```diff
@@ -756,7 +735,8 @@ authors:
   - Supervision
   - Writing – Review & editing
 - github: jharenza
-  name: Jo Lynne Rokita
+  symbol_str: ^
+  name: Jo Lynne Rokita<sup>*</sup>
```
Comment (jaclyn-taroni) [inline content/metadata.yaml:739]: This broken things
```suggestion
  name: Jo Lynne Rokita
```

**Comment by jaclyn-taroni** [comment]
I'm going to try and push something

**Comment by jharenza** [comment]
you got it!

**Review by jaclyn-taroni — APPROVED** [review]

**Pull request merged**